CONTRIBUTING.md

```diff
@@ -1,35 +1,60 @@
 # Contributing to the Parse JavaScript SDK
 We want to make contributing to this project as easy and transparent as possible.
 
-If you're looking to get started, but want to ease yourself into the codebase, look for issues tagged [good first bug](https://github.com/ParsePlatform/Parse-SDK-JS/labels/good%20first%20bug). These are simple yet valuable tasks that should be easy to get started.
+If you're looking to get started, but want to ease yourself into the codebase, look for issues tagged [good first bug](https://github.com/parse-community/Parse-SDK-JS/labels/good%20first%20bug). These are simple yet valuable tasks that should be easy to get started.
```

Comment:
There isn't a `good first bug` label there is a `good first task`

https://github.com/parse-community/Parse-SDK-JS/labels/good%20first%20task